src/EnergyPlus/SurfaceGeometry.cc

```diff
@@ -12660,12 +12660,26 @@ namespace SurfaceGeometry {
         }
     }
 
-    void SetupRadiantEnclosuresAndAirBoundaries(bool &ErrorsFound)
+    void SetupEnclosuresAndAirBoundaries(Array1D<DataViewFactorInformation::ZoneViewFactorInformation> &Enclosures, // Radiant or Solar Enclosures
+                                         SurfaceGeometry::enclosureType const &EnclosureType,                       // Radiant or Solar
+                                         bool &ErrorsFound)                                                         // Set to true if errors found
```

Comment:
Replace two very similar functions `SetupRadiantEnclosuresAndAirBoundaries` and `SetupSolarEnclosuresAndAirBoundaries` with a single function that's called twice, `SetupEnclosuresAndAirBoundaries`.